src/naf-dialog-adapter.js

```diff
@@ -983,7 +978,7 @@ export default class DialogAdapter {
         }
 
         if (!this._reconnectionTimeout) {
-          this._reconnectionTimeout = setTimeout(() => this.reconnect(), this._reconnectionDelay);
+          this._reconnectionTimeout = setInterval(() => this.reconnect(), this._reconnectionDelay);
```

Comment:
Are you sure about this change? This seems weird to use setInterval here. You don't use the updated reconnectionDelay in this case to retry with more delay for each try. You should put back setTimeout here and clearTimeout at https://github.com/mozilla/hubs/blob/ad709d3f367d9267ef54266b076bd731aeb4522a/src/naf-dialog-adapter.js#L959

Reply:
Good catch @vincentfretin! Actually we were  just failing without retrying. I've updated so the retry mechanism works and we keep retrying until the connection timeout in https://github.com/mozilla/hubs/blob/f180d53b4dda27a6f239a65e6220978099286040/src/hub.js#L633 expires, which is 90secs now.